shared/javascripts/network_service.js

```diff
@@ -437,15 +437,19 @@ var privlyNetworkService = {
    * Show/hide the appropriate navigation items for when the user is logged out.
    */
   showLoggedOutNav: function() {
-    
-    // Don't show the nav at all if the content is injected.
-    if(typeof privlyHostPage !== "undefined" && privlyHostPage.isInjected()) {
+  var isAndroid=privlyNetworkService.platformName();
```

Comment:
inconsistent block indentation. Also, if there are tabs in this, you should remove them.